docs/helm-chart/index.rst

```diff
@@ -117,6 +128,12 @@ To uninstall/delete the ``airflow`` deployment:
 
     helm delete airflow --namespace airflow
 
+**Explanation**
+- `helm delete` is a command to delete a release (it was renamed to `helm uninstall`)
+- the first 'airflow' argument is the release name
+- the second 'airflow' argument is the namespace
+
+
```

Comment:
instead, can you amend existing to replace delete with uninstall?  

also, rather than explaining this
```
- the first 'airflow' argument is the release name
- the second 'airflow' argument is the namespace
```

maybe better would be to choose better names in the example in the first place... so instead of `airflow` (namespace) choose `my-namespace`, and instead of `airflow` (release) maybe choose `my-release-name` or `my-airflow-release`.... WDYT?

Reply:
I like this idea. It helps break up the Helm commands to identify different things with different names, instead of leaving that implicit. I would try to identify things with their types, so something like `my-airflow-namespace`, even though that looks a little wordy, it's more helpful for people who aren't familiar with it.

Reply:
Another thing that might help is using (making up a term here) the "options assignment" syntax, if Helm supports it:
```bash
helm repo add apache-airflow https://airflow.apache.org
helm upgrade --install=airflow apache-airflow/airflow --create-namespace --namespace=airflow
helm upgrade airflow apache-airflow/airflow --namespace=airflow
helm delete airflow --namespace=airflow
```

But if it doesn't then feel free to disregard.

Reply:
Yeah that's a good idea @blag but unfortunately I don't think that the helm command allows that... it's just positional

I think I found this confusing at one point in the past also